src/cli/commands/deploy.ts

```diff
@@ -31,6 +31,8 @@ export default function registerCommand(program: Command) {
     .option("--app-location <path>", "the folder containing the source code of the front-end application", DEFAULT_CONFIG.appLocation)
     .option("--api-location <path>", "the folder containing the source code of the API application", DEFAULT_CONFIG.apiLocation)
     .option("--output-location <path>", "the folder containing the built source of the front-end application", DEFAULT_CONFIG.outputLocation)
+    .option("--api-language <apiLanguage>", "the runtime language of the function")
```

Comment:
Just caught that it's also missing default values and environment variables, could you add it?

Reply:
+1. @sgollapudi77  Lets include those changes here.

Reply:
Added these now.